### ISSUE TITLE
hauser doesn't build: "undefined:shouldRetry"

### ISSUE THREAD

**Issue by radix** [fullstorydev/hauser#11]
Hi, I can't get hauser to compile given the instructions in the README. I'm using Go 1.10.

```
radix@pop-os:~$ GOOS=linux GOARCH=amd64 go get github.com/fullstorydev/hauser
# github.com/fullstorydev/hauser/vendor/cloud.google.com/go/storage
go/src/github.com/fullstorydev/hauser/vendor/cloud.google.com/go/storage/invoke.go:31:6: undefined:shouldRetry
```

The error is in the google cloud package, which makes me think that the vendored libs got messed up, or perhaps doesn't have thorough enough coverage and maybe something is still being downloaded from a master branch?

**Comment by sanjaybv** [comment]
@radix Hey! Hauser is using an older version of the google cloud package and the go file in the package `cloud.google.com/go/storage` that defines the function `shouldRetry` (for go 1.10) is in `go110.go.pending`. To make that function available for go 1.10, you'll have to rename the file `go110.go.pending` to `go110.go` and compile again.

I'll add a note in our system to update this package. For now, I believe the above solution will make it work again. Thanks for letting us know!

**Comment by jameremo** [comment]
@radix In the interim since this issue was opened, we've upgraded our packages and we should be building on go 1.10 and 1.11 now. I'm going to close the issue, but please feel free to reopen if you need anything else.
